Documentation/ceph-examples.md

```diff
@@ -74,3 +82,5 @@ See the [Object Store CRD](ceph-object-store-crd.md) topic for more details on t
 
 ### Object Storage User
 - `object-user.yaml`: Creates a simple object storage user and generates creds for the S3 API
+
```

Comment:
nit: remove extra lines at the end of the file?

Reply:
Done